Sources/TensorFlow/Loss.swift

```diff
@@ -262,6 +282,8 @@ func _vjpSoftmaxCrossEntropyHelper<Scalar: TensorFlowFloatingPoint>(
 }
 
 /// Returns the sigmoid cross entropy (binary cross entropy) between logits and labels.
+/// Given the logits and probabilites, the sigmoid cross entropy computes `reduction(-sigmoid(logits) * log(p))`
+/// Where sigmoid(x) = `1 / (1 + exp(-x))`
```

Comment:
Will equations like these be shown as LaTex does?

Reply:
No, code enclosed in backticks appear with inline code formatting, just like in Markdown.
The [Jazzy API documentation generator](https://github.com/realm/jazzy#linux) doesn't support LaTex formatting.

Reply:
I think having no LaTex—at least for now—is good for user experience. @fchollet wrote an entire (best-selling) [book](https://www.manning.com/books/deep-learning-with-python) on deep learning without any LaTex: 

> "...we’ll steer away from mathematical notation, which can be off-putting for those without any mathematics background and isn’t strictly necessary to explain things well."

Reply:
@8bitmp3 you just made me feel nervous 😬 because my book will include equations.

Reply:
![](https://media.giphy.com/media/11F0d3IVhQbreE/giphy.gif)

#suntorytime